README.md

```diff
@@ -1,2 +1,279 @@
+  "request_medium": "<<web | android | ios>>",
+  "sdk_name": "1",
+  "data": {
+    "metadata": {
+      "acc_number": "<<account_number>>",
+      "start_date": "13/02/2019",
+      "end_date": "12/02/2020"
+    },
+    "transactions": [
+      {
+        "date": "2019-03-17",
+        "value_date": "2019-03-17",
+        "chq": "",
+        "particulars": "UPI/123123123/From:upi@ybl/To:upi@ybl/Payment from PhonePe",
+        "balance": 100,
+        "amount": 100,
+        "label": "CREDIT",
+        "validation": true
+      },
+      {
+        "date": "2019-03-17",
+        "value_date": "2019-03-17",
+        "chq": "907632321560",
+        "particulars": "PCA:123123123:123123123:GOOGLE *SERVICES       g.co/helppay# CA",
+        "balance": 99,
+        "amount": 1,
+        "label": "DEBIT",
+        "validation": true
+      }
+    ]
+  }
+}
+```
+<a name="bsaErrorWebhookReqBody"></a>
+#### 5.2 FAILURE REQUEST BODY
+
+```json
+{
+  "id": "<<transaction_id>>",
+  "mode": "REDIRECT",
+  "env": "PRODUCTION",
+  "bank": "SBI",
+  "response_code": 0,
+  "response_message": "Transaction Failure",
+  "last_user_stage_code": "<<Stage Code>>",
+  "last_user_stage": "<<Stage>>",
+  "request_version": "1.0",
+  "request_medium": "<<web | android | ios>>",
+  "sdk_name": "1",
+  "error": {
+    "code": "<<error_code>>",
+    "message": "<<error_message>>"
+  }
+}
+```
+<a name="bsaErrorCodeWebhook"></a>
+#### 5.3 ERROR CODES AND MESSAGES
+
+| Code  | Message |
+|---|---|
+| 651 | Technical Error |
+| 652 | Session Closed |
+| 653 | Bank Server Unresponsive Error |
+| 654 | Consent Denied Error |
+| 655 | Document Parsing Error |
+| 656 | Validity Expiry Error |
+| 657 | Authentication Error |
+| 658 | No Entry Found Error |
+| 659 | Stage Timeout Error |
+| 660 | Session Closed On Retry |
+| 670 | No Session Found Error |
+| 671 | Error Not Recorded |
+
+<a name="bsaStage"></a>
+### 6. STAGE
+
+After creating an initialization request successfully you can check at which stage your transaction is currently.
+
+    GET: {{base_url}}/bsa/v1/stage/<<transaction_id>>
+
+<a name="bsaStageSuccess"></a>
+#### 6.1 SUCCESSFUL RESPONSE BODY
+
+```json
+{
+    "id": "<<transaction_id>>",
+    "mode": "REDIRECT",
+    "env": "PRODUCTION",
+    "request_version": "1.0",
+    "bank": "SBI",
+    "last_user_stage": "login_opened",
+    "last_user_stage_code": "LO"
+}
+```
+
+<a name="bsaUserStage"></a>
+#### 6.2 USER STAGES
+
+| Code  |  Stage |
+|---|---|
+| LO | login_opened | 
+| F | failed | 
+| LS | login_success | 
+| AS | account_selected | 
+| SD | statement_downloaded | 
+| SP | statement_parsed | 
+| TS | transaction_successful | 
+| TI | transaction_initiated | 
+| VC | validated_captcha | 
+| VO | validated_otp | 
+| VS | validated_security_answer |
+
+<a name="bsaStageFailure"></a>
+#### 6.3 FAILURE RESPONSE BODY
+```json
+{
+    "statusCode": 404,
+    "errors": [],
+    "message": "transaction id not found in our records"
+}
+```
+
```

Comment:
Mention Technical support URL here. 

In case you are facing any issues with integration please open a ticket on our [support portal](#link here)